docs/pages/database-access/guides/cockroachdb-self-hosted.mdx

```diff
@@ -84,17 +97,17 @@ Make sure to assign the user proper permissions within the database cluster.
 Refer to [Create User](https://www.cockroachlabs.com/docs/stable/create-user.html)
 in Cockroach docs for more information.
 
-### Setup mutual TLS
+### Set up mutual TLS
 
-To setup mutual TLS authentication, you need to make sure that:
+To set up mutual TLS authentication, you need to make sure that:
 
 - Teleport trusts certificates presented by CockroachDB nodes.
 - CockroachDB trusts client certificates signed by Teleport.
 
 Generate the secrets by running the following `tctl` command against your
 Teleport cluster:
 
-```code
+```bash
```

Comment:
I thought we're using the new `code` type for these snippets now? Since it enables copy.

Same throughout.

Reply:
The `code` snippet doesn't render the escaped line breaks, so everything is shown on one line. Since there is only one command to copy in each of these snippets, and readers can still copy the snippet via the button on the upper right, I thought it would make sense to preserve the escaped line breaks. 

I can create an issue related to the escaped line breaks and we can change all `bash` blocks to `code` blocks once that's resolved, if that works for you!